### PR TITLE
react-native-sdk: add swarm connect to ipfs api

### PR DESCRIPTION
**Pull request by flyskywhy**
Add `swarm connect` to [Update React Native APIs to match more closely the JS and Rest API](https://github.com/textileio/react-native-sdk/issues/108)

Need PR:
[android-textile: add swarm connect to ipfs api](https://github.com/textileio/android-textile/pull/64)  and set android-textile to v2.0.9 
[ios-textile: add swarm connect to ipfs api](https://github.com/textileio/ios-textile/pull/84) and set ios-textile to v2.0.11

### CODE REVIEW

**Comment by flyskywhy** [comment]
@asutula , since android-textile has been set to v2.0.9 and ios-textile has been set to v2.0.11 , it's possible to merge this PR :stuck_out_tongue_closed_eyes:

File: android/src/main/java/io/textile/rnmobile/IpfsBridge.java [hunk 1]
```diff
@@ -39,6 +39,25 @@ public void run() {
         });
     }
 
+    @ReactMethod
+    public void connect(final String multiaddr, final Promise promise) {
```
Comment (asutula) [inline android/src/main/java/io/textile/rnmobile/IpfsBridge.java]: Can we just update this to `swarmConnect` everywhere for consistency with the native sdks?

Comment (flyskywhy) [reply android/src/main/java/io/textile/rnmobile/IpfsBridge.java]: I just named it for consistency with the js-http-client sdk , ref to https://github.com/textileio/js-http-client/blob/5f8b3a421ca682b3ba5ed3d4af96bca1d0913a39/src/modules/ipfs.ts#L58 ... So should we still update this to swarmConnect?

Comment (asutula) [reply android/src/main/java/io/textile/rnmobile/IpfsBridge.java]: Ah ok, that makes sense. Let's just leave this as is then.

**Comment by asutula** [comment]
Ok 3.0.13 is released!

**Pull request merged**